libs/ui/src/hooks/auth/use_dipped_smart_card_auth.ts

```diff
@@ -0,0 +1,44 @@
+import useInterval from 'use-interval';
+import { DippedSmartCardAuth } from '@votingworks/types';
+import { useEffect, useState } from 'react';
+
+const AUTH_STATUS_POLLING_INTERVAL = 100;
```

Comment:
I lowered the polling interval from 200 to 100 since we temporarily have two layers of polling: VxAdmin frontend ➡️ VxAdmin backend, and VxAdmin backend ➡️ card reader. And if they're offset in just the right way, the times can add up.

In my JavaCard implementation of backend auth, I'm hoping to avoid the second layer (VxAdmin backend ➡️ card reader) by using listeners